integration/test/Testlib/Env.hs

```diff
@@ -99,7 +99,7 @@ mkGlobalEnv cfgFile = do
         gDomain1 = intConfig.backendOne.originDomain,
         gDomain2 = intConfig.backendTwo.originDomain,
         gDynamicDomains = (.domain) <$> Map.elems intConfig.dynamicBackends,
-        gDefaultAPIVersion = 5,
+        gDefaultAPIVersion = 6,
```

Comment:
A good catch! How did you find it? By running tests or by some other means? Let's add this to the version bump checklist!

Reply:
I looked at the previous PR for finalizing V4.